libraries/Bluefruit52Lib/src/BLEPeriph.h

```diff
@@ -50,6 +51,7 @@ class BLEPeriph
     bool    connected(uint16_t conn_hdl); // Connected as prph to this connection
     uint8_t connected(void);              // Number of connected as peripherals
 
+    bool getBonds(bonded_device_info* bondedPeers, uint32_t bondedPeersSize, uint32_t* actualBondedPeersSize);
```

Comment:
yeah, I would also want to expose the bond list, however I would prefer to retrieve one by one as an array than returning the whole list which can be long and application probably will process one by one anyway afterwards. Or I am missing something, could you tell me about your usage.

Reply:
I took your suggestion and removed this portion of this PR. I'll consider the addition a bit more and if it still makes sense to add I will create a separate PR later to focus on that change by itself.

Reply:
Yeah, it make sense to retrieve data from bond list, there is bond management service which can benefit from this as well, I planed to do this but haven’t got time to do so.
- https://github.com/adafruit/Adafruit_nRF52_Arduino/issues/374 

the separate PR makes sense. It is best to focus on each feature per PR to make it easier to back-tracking when fixing bugs. We could piggy-pack multiple features if they are related though.